bundles/org.openhab.core.config.discovery/src/main/resources/OH-INF/config/inbox.xml

```diff
@@ -8,16 +8,12 @@
 	<config-description uri="system:inbox">
 		<parameter name="autoIgnore" type="boolean">
 			<label>Auto Ignore</label>
-			<description><![CDATA[
-				<p>If enabled, an inbox result matching an existing thing is automatically ignored.</p>
-				<p>If set to false, the inbox may contain results although identical things already exist.</p>
-			]]>
-			</description>
+			<description><![CDATA[If enabled, an Inbox result matching an existing thing is automatically ignored.<br/>If set to false, the Inbox may contain results although identical things already exist.]]></description>
```

Comment:
Is the `<![CDATA[` still used here so you don't have to escape HTML tags?

I had a look at the webui translations and saw that it uses `<br>` instead of `<br/>`:

https://github.com/openhab/openhab-webui/blob/7c8e7875832344edf335f4095c3a2d6f847cf276/bundles/org.openhab.ui/web/src/assets/i18n/empty-states/en.json#L6

It would make it a bit more consistent if we would use the same tags everywhere. I would prefer `<br>` since it is shorter, WDYT @ghys? It seems that `<br />` (with space) should be used to be XHTML compliant.

Reply:
Both are fine in HTML5, I tend to agree with https://stackoverflow.com/a/4288474 although depending on the context/surrounding code I sometimes use `<br />`. In parameter descriptions I think `<br>` is better since it prevents escaping issues.

Reply:
Yes, I left some Character Data stuff because of the escaping. The `<description>` tag is a simple type element.and does not allow other HTML/XML Tags inside.

Talking about labels and descriptions something else came to my mind. When using the i18n from properties files the values inside the XML are kind of useless. One day someone will.be confused by the double place to maintain the labels.or description. Because of that I thought it might be better to switch every config description to.use custom keys like `@text/<key>` which separates the definition from i18n logic and prevents a lot of work in the future.

I will rework this PR accordingly.

Reply:
I will not change it. Once we have a proper XSLT transformation the problem will be solved too.

I changed the `<be>` tags.